Sources/Networking/HTTPClient/HTTPClient.swift

```diff
@@ -92,6 +93,41 @@ class HTTPClient {
         return self.systemInfo.responseVerificationMode.isEnabled
     }
 
+    // Visible for tests
+    var defaultHeaders: RequestHeaders {
+        var headers: RequestHeaders = [
+            "content-type": "application/json",
+            "X-Version": SystemInfo.frameworkVersion,
+            "X-Platform": SystemInfo.platformHeader,
+            "X-Platform-Version": SystemInfo.systemVersion,
+            "X-Platform-Flavor": self.systemInfo.platformFlavor,
+            "X-Client-Version": SystemInfo.appVersion,
+            "X-Client-Build-Version": SystemInfo.buildVersion,
+            "X-Client-Bundle-ID": SystemInfo.bundleIdentifier,
+            "X-StoreKit2-Enabled": "\(self.systemInfo.storeKit2Setting.isEnabledAndAvailable)",
+            "X-Observer-Mode-Enabled": "\(self.systemInfo.observerMode)",
+            RequestHeader.sandbox.rawValue: "\(self.systemInfo.isSandbox)"
+        ]
+
+        if let storefront = self.systemInfo.storefront {
+            headers["X-Storefront"] = storefront.countryCode
```

Comment:
FFTI but in android we add all the headers to the same map and then filter out null values. That might be slightly more legible?